start

```diff
@@ -147,17 +147,17 @@ function process_args() {
       ENABLE="$1"
       shift
       ;;
-    --enable-soroban-rpc)
-      ENABLE_SOROBAN_RPC=true
+    --enable-stellar-rpc)
+      ENABLE_STELLAR_RPC=true
```

Comment:
Can we retain the `--enable-soroban-rpc` option for backwards compat?

We don't need to add `--enable-stellar-rpc` because we've moved to using the `--enable` option instead, so folks can do `--enable rpc` already.